pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -564,6 +574,39 @@ func publishMetrics(ctx *zedagentContext, iteration int) {
 			ReportAppMetric.Disk = append(ReportAppMetric.Disk,
 				appDiskDetails)
 		}
+
+		acMetric := lookupAppContainerMetric(ctx, aiStatus.UUIDandVersion.UUID.String())
+		// upload acMetric when it's been newly updated
```

Comment:
Isn't that going to result in holes in the time series? Please check with @bharani-zededa

Reply:
Talked to @bharani-zededa , he said the 'containerStats' pointer inside 'appMetric' is null is fine. the cloud UI default is 1 hour, so it's just less samples in 1 hour (in this case, 12 samples per hour)